core/templates/core/announcement_email.html

```diff
@@ -0,0 +1,9 @@
+{% autoescape off %}
+Below is an announcement from {{user}}:
+
+{{body}}
+
+Best,
```

Comment:
Let's take this `Best, {{user}}` thing out, I think writing a salutation for them would be a little strange in use